### ISSUE TITLE
Chromedriver Default Port?

### ISSUE THREAD

**Issue by zoracon** [Hainish/apk-downloader#1]
What: 
When I run the program it halts to tell me Chromedriver needs to run on port 4444 specifically.

```
Waiting...
Downloading [redacted]...
thread 'main' panicked at 'chromedriver must be running on port 4444', /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/apk-downloader-0.2.1/src/main.rs:175:19
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

"Fix": By default, chromedriver ran on port 9515 for me. I am not sure if that is due to a previous configuration for me, or that is the true default. I ran `chromedriver --port=4444` and that allowed the program to run normally.

**Comment by zoracon** [comment]
Oh! I also used APK Pure obviously to run into this.

**Comment by zoracon** [comment]
> What:
> When I run the program it halts to tell me Chromedriver needs to run on port 4444 specifically.
> 
> ```
> Waiting...
> Downloading brightwheel...
> thread 'main' panicked at 'chromedriver must be running on port 4444', /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/apk-downloader-0.2.1/src/main.rs:175:19
> note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
> ```
> 
> "Fix": By default, chromedriver ran on port 9515 for me. I am not sure if that is due to a previous configuration for me, or that is the true default. I ran `chromedriver --port=4444` and that allowed the program to run normally.

**Comment by Hainish** [comment]
Yes, probably a good idea to make this the default port.  Fixed in cd188d89ba40af646bc20696598f8943f4ba76e2
